sdk/va-report-components/documentation/docs/api/ReportHandle.md

```diff
@@ -146,3 +157,20 @@ Refreshes the data for all of the objects in the report.
 ### reloadReport(): void
 
 Reloads the report. This updates all report content and data, which resets all filters and parameters to their default values.
+
+### addEventListener(eventType: string, listener: (event: Object) => void)
+
+Adds an event listener to the `ReportHandle` to call the supplied listener when the specified event occurs.
+
+#### Arguments
+
+`eventType` is a string that represents the event type to listen for. These event types are supported:
```

Comment:
Unless I'm missing something, there's only one event type listed here. The lead-in phrase should be: This even type is supported:

Reply:
Yes, there's only one, but there will likely be more in the future. I was going off of the way we structured this event type list [when we first added event types to ObjectHandle](https://github.com/sassoftware/sas-viya-sdk-js/pull/24/files#diff-5112e831a328a8198f410c3e233116ea0bf45c5d94e77caa1eb7c59e92dc2619R67) (there was only one then, too) & matching [the way we currently structure the list for ObjectHandle](https://github.com/sassoftware/sas-viya-sdk-js/pull/60/files#diff-5112e831a328a8198f410c3e233116ea0bf45c5d94e77caa1eb7c59e92dc2619L84) now that there is more than one type there. 

I can definitely change it to "This event type is supported" but just wanted to make sure you still want the change given that info!

Reply:
@eheckel, since you'll be adding more types in the future, let's just leave it as is.